src/test/java/fi/otavanopisto/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -0,0 +1,218 @@
+import fi.metatavu.ptv.client.model.V7VmOpenApiOrganization;
+import fi.metatavu.ptv.client.model.V7VmOpenApiService;
+import fi.metatavu.ptv.client.model.V7VmOpenApiServiceServiceChannel;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.servicechannels.ServiceChannelType;
+
+public class PtvDownload {
+  
+  private static final String PTV = "v7";
+
+  public static void main(String[] args) throws JsonParseException, JsonMappingException, IOException {
+    new PtvDownload()
+      .downloadOrganizations()
+      .printIds();
+  }
+  
+  private void printIds() throws JsonParseException, JsonMappingException, IOException {
+    File[] organizationFiles = listJsonFiles(getOrganizationsFolder());
+    File[] serviceFiles = listJsonFiles(getServicesFolder());
+    File[] serviceChannelFiles = listJsonFiles(getServiceChannelsFolder());
+    
+    printIdList("ORGANIZATIONS", organizationFiles);
+    printIdList("SERVICES", serviceFiles);
+    printIdList("SERVICE_CHANNELS", serviceChannelFiles);
+    
+    for (ServiceChannelType type : ServiceChannelType.values()) {
+      printServiceChannelList(type, serviceChannelFiles);
+    }
+  }
+  
+  private ServiceChannelType resolveChannelType(String type) {
+    switch (type) {
+      case "EChannel":
+        return ServiceChannelType.ELECTRONIC_CHANNEL;
+      case "ServiceLocation":
+        return ServiceChannelType.SERVICE_LOCATION;
+      case "PrintableForm":
+        return ServiceChannelType.PRINTABLE_FORM;
+      case "Phone":
+        return ServiceChannelType.PHONE;
+      case "WebPage":
+        return ServiceChannelType.WEB_PAGE;
+      default:
+        return null;
+    }
+  }
+
+  private void printIdList(String constName, File[] files) {
+    List<String> ids = new ArrayList<>();
+    for (File file : files) {
+      ids.add(String.format("      \"%s\"",  StringUtils.stripEnd(file.getName(), ".json")));
+    }
+    
+    System.out.println(String.format("\n    public final static String[] %s = {", constName));
+    System.out.println(StringUtils.join(ids, ",\n"));
+    System.out.println("    };");
+  }
+
+  private void printServiceChannelList(ServiceChannelType type, File[] files) throws JsonParseException, JsonMappingException, IOException {
+    List<String> ids = new ArrayList<>();
+    for (File file : files) {
+      ServiceChannelTypeExtract serviceChannelTypeExtract = getObjectMapper().readValue(file, ServiceChannelTypeExtract.class);
+      if (resolveChannelType(serviceChannelTypeExtract.getServiceChannelType()) == type) {
+        ids.add(String.format("      \"%s\"",  StringUtils.stripEnd(file.getName(), ".json")));
+      }
+    }
+    
+    System.out.println(String.format("\n    public final static String[] %s_SERVICE_CHANNELS = {", type.name()));
+    System.out.println(StringUtils.join(ids, ",\n"));
+    System.out.println("    };");
+  }
+
+  private PtvDownload downloadOrganizations() throws JsonParseException, JsonMappingException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)